src/constants/routes.js

```diff
@@ -1,137 +1,89 @@
-import AddBookmark from '@screens/bookmarks/addBookmark';
```

Comment:
This file does not import any React components anymore and is truly a constants file. So, whenever a test imports `@constants`, no components will be included in the code executed by jest.